CMakeLists.txt

```diff
@@ -1,155 +1,143 @@
 ### CMakeList for MC Hydra
 
-cmake_minimum_required(VERSION 3.0)
-cmake_policy(SET CMP0054 OLD)
```

Comment:
Do not set a policy to OLD unless you have to! CMake will drop the old versions of policies at some point. (I think this one is being dropped in 3.11). This is for quoted if statements and the old behavior is very confusing for most users. I did not see any remaining if statements that would be affected by this.

Reply:
Did you tested what happens removing this ?

Reply:
Yes, I built all four backends on Goofy with this PR.

Reply:
If I remove the line -cmake_policy(SET CMP0054 OLD) , I get this from cmake:

```
 CMake Warning (dev) at /usr/share/cmake/Modules/FindOpenMP.cmake:200 (if):
  Policy CMP0054 is not set: Only interpret if() arguments as variables or
  keywords when unquoted.  Run "cmake --help-policy CMP0054" for policy
  details.  Use the cmake_policy command to set the policy and suppress this
  warning.

  Quoted variables like "c" will no longer be dereferenced when the policy is
  set to NEW.  Since the policy is not set the OLD behavior will be used.
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindOpenMP.cmake:324 (_OPENMP_GET_FLAGS)
  CMakeLists.txt:70 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.
 ```

How do you interpret this?

Reply:
That is because you set the minimum version to 3.0. You should set it to 3.1. (3.0 was not used heavily, there were lots of fixes in 3.1)

Reply:
So, please promote it to 3.1.

Reply:
It is 3.1 in this PR. :)